test/jdk/java/awt/print/PrinterJob/PrintAllFonts.java

```diff
@@ -24,6 +24,7 @@
 /**
  *
  * @bug 4884389 7183516
+ * @key printer
  * @summary Font specified with face name loses style on printing
  * @run main/manual PrintRotatedText
```

Comment:
Is it intentional that this test presumably runs another test — `PrintRotatedText` — instead?